internal/cli/cli.go

```diff
@@ -119,6 +121,10 @@ func (c *cli) configureRenderer() {
 	if c.json {
```

Comment:
**Request:** I think it would be nice to add validation to prevent using `--json` and `--csv` together.

Reply:
I think this is where a `--format=csv` or `--format=json` might serve us better: conflict management on format is difficult the more formats we add. The `--csv` flag assumes you didn't do a `--json` and vice versa.

Reply:
This was actually a conscious decision w/ v1 to eliminate the `--format` flag because there was only one available format at the time and even with two options, two separate flags seemed more ergonomic. If we were to support more formats (ex: tsv, yaml, etc), then it would make sense to re-introduce the `--format` flag.